### PR TITLE
Gutter Gang

### PR DESCRIPTION
**Pull request by toomykins**
videogames

:cl:
 * rscadd: adds gutter gang
 * rscdel: dignity

### CODE REVIEW

**Comment by Reviire** [comment]
Literally how did you fuck it up

File: code/datums/religions.dm [hunk 1]
```diff
@@ -1036,3 +1036,15 @@
...
+	name = "Gutter Gang"
+	deity_name = "The Gutter Gods"
+	bible_names = list("Toomykins' admin application", "The Splutterings of the Gutterings", "The Brown Jumpsuit ft. Gary", "A guide to proxy warfare")
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Gutter Gangmember"
+	female_adept = "Gutter Gangmember"
+	keys = list("gutter", "toomykins", "shitter")
+
+/datum/religion/gutter/equip_chaplain(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/under/color/brown(H), slot_w_uniform)
```
Comment (MadmanMartian) [inline code/datums/religions.dm:1050]: this under subtype doesn't exist.

Comment (toomykins) [reply code/datums/religions.dm:1050]: i dont understand why it works in assistant.dm but not here

**Comment by Exxion** [comment]
References to players are bad
References to players made *by said players* are very bad

**Comment by strawberrytan** [comment]
(you)

**Pull request closed without merging**